WordPress/gutenberg
# DropdownMenu v2: Fix submenu chevron direction in RTL languages

## What?

Make sure the RTL styles are beeing applied for the submenu trigger item.

## Why?

The arrows should point the other way for RTL languages.

## How?

Fix how RTL styles are added.

## Testing Instructions
1. Open up storybook and navigate to the dropdown v2 component.
2. Enable RTL
3. Confirm the arrows point to the left on the submenu trigger item.

## Screenshots or screencast 
| Before | After |
|-----|-----|
| <img width="560" alt="image" src="https://github.com/WordPress/gutenberg/assets/1415747/bbdde52a-5abc-453f-bafa-8f7959c35d18"> | <img width="560" alt="image" src="https://github.com/WordPress/gutenberg/assets/1415747/a8179c93-c37c-411b-9ac6-6bf54f2aa4a7"> |
